config.js

```diff
@@ -1306,6 +1306,9 @@ var config = {
     //     hideJoinRoomButton: false,
     // },
 
+    // When false, virtual background feature will be disabled.
+    // enableVirtualBackground: true,
```

Comment:
Look at the rest of the options. The default value is false, so it should be names disableVirtualBackground.

Reply:
Changed